packages/oui-select/README.md

```diff
@@ -68,6 +68,39 @@
 </oui-select>
 ```
 
+### Disabled Items
+
+#### Using Basic Array
+
+```html:preview
+<oui-select name="letter"
+    model="$ctrl.letter"
+    data-title="Select a letter"
+    placeholder="Select a letter..."
+    items="['a', 'b', 'c']"
+    disabled-items="['b']"
+    required
+    data-align="start">
+    <span ng-bind="$item"></span>
+</oui-select>
+```
+
+#### Using Object Array
+
+```html:preview
+<oui-select name="country"
+    model="$ctrl.country"
+    data-title="Select a country"
+    placeholder="Select a country..."
+    items="$ctrl.countries"
+    disabled-items="['Algeria', 'Canada', 'France', 'United States']"
```

Comment:
I think that you should add a note under/above these examples about the fact that `disabled-items` need an array of values based on the match :)

Since `match` is optional, how will work this property in the case of the property is not defined ?
This should be documented too.